47.md

```diff
@@ -38,21 +41,26 @@ Request:
 Response:
 ```jsonc
 {
-    "status": "ok", // status, "ok" | "error"
-    "event": "0123456789abcdef...", // event the command is in response to, string
-    "data": { // response data
+    "result_type": "pay_invoice", //indicates the structure of the result field
```

Comment:
The event ID of the request is already in the `e` tag.

Reply:
I put it in because I thought that it was something that @vitorpamplona asked. So you can type-cast the result object in the right class without needing to look up the request by id to check what type it had.

Reply:
But indeed, if requests are all stored in memor then I guess it's no big deal to look up the request type by the id.

Reply:
It's not redundant. Most JSON parsing tools use a static type to identify which object it should create. Figuring out the reply type by event id is not ideal.